api/opencl_platform_layer.asciidoc

```diff
@@ -1444,13 +1444,22 @@ include::{generated}/api/version-notes/CL_DEVICE_GENERIC_ADDRESS_SPACE_SUPPORT.a
 // This query didn't exist pre-OpenCL 3.0, but:
 //Support for the generic address space is required for an OpenCL 2.0, 2.1, or 2.2 device.
 
-| {CL_DEVICE_DEVICE_ENQUEUE_SUPPORT_anchor}
+| {CL_DEVICE_DEVICE_ENQUEUE_CAPABILITIES_anchor}
 
-include::{generated}/api/version-notes/CL_DEVICE_DEVICE_ENQUEUE_SUPPORT.asciidoc[]
-  | cl_bool
-      | Is {CL_TRUE} if the device supports device-side enqueue and on-device queues, and {CL_FALSE} otherwise.
+include::{generated}/api/version-notes/CL_DEVICE_DEVICE_ENQUEUE_CAPABILITIES.asciidoc[]
+  | cl_device_device_enqueue_capabilities
+      | Describes device-side enqueue capabilities of the device.
+        This is a bit-field that describes one or more of the following
+        values:
+
+        {CL_DEVICE_QUEUE_SUPPORTED_anchor} - Device supports device-side enqueue and On-Device queues
+
+        {CL_DEVICE_QUEUE_REPLACEABLE_DEFAULT_anchor} - Device supports a replaceable default On-Device queue
```

Comment:
The query is `CL_DEVICE_DEVICE_ENQUEUE_CAPABILITIES` and the bitfield type is `cl_device_device_enqueue_capabilities`, but the bitfield values are `CL_DEVICE_QUEUE_SUPPORTED` etc.

I.e. it's device-enqueue/device-enqueue/device-queue.   Should this be consistent?  Achieved by either by changing these values to the form `CL_DEVICE_ENQUEUE_SUPPORTED`, or by changing the query and type names.

Changing the query name ends up sounding very similar to `CL_DEVICE_QUEUE_ON_DEVICE_PROPERTIES`, which is maybe good because they are similar, but the distinction between capabilities being an immutable aspect of the implementation and properties being aspects that can be enabled is pretty subtle.

Is everyone is happy with the naming as-is then I'm happy to just resolve this comment.